ctapipe/reco/tests/test_HillasReconstructor.py

```diff
@@ -261,13 +260,32 @@ def test_CameraFrame_against_TelescopeFrame(filename):
             for field, cam in result_camera_frame.items():
                 tel = getattr(result_telescope_frame, field)
 
+                kwargs = dict(rtol=6e-3, equal_nan=True)
```

Comment:
a reason for `6e-3`?

Reply:
that was needed to make the tests pass... It's more a regression test for the future.